build.gradle

```diff
@@ -484,7 +490,16 @@ project('standalone') {
     applicationDefaultJvmArgs = ["-server", "-Xmx4g", "-XX:+HeapDumpOnOutOfMemoryError",
                                  "-Dlogback.configurationFile=PRAVEGA_APP_HOME/conf/logback.xml",
                                  "-Dlog.dir=PRAVEGA_APP_HOME/logs",
-                                 "-Dlog.name=pravega"]
+                                 "-Dlog.name=pravega",
+                                 "-Xdebug",
+                                 "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=y"]
+    startScripts {
+        classpath += files('./pluginlib')
+        doLast {
+            def scriptFile = file getUnixScript()
+            scriptFile.text = scriptFile.text.replace('$APP_HOME/lib/pluginlib', '$APP_HOME/pluginlib/*')
```

Comment:
Minor detail:  does the controller create `pluginlib`? If so when?

Reply:
Updated the `installDist` to create the dir.